VATSpy.dat

```diff
@@ -10983,6 +10983,47 @@ SBVC|Vitoria da Conquista|-14.90777778|-40.91472222|VDC|SBRE|0
 SBVG|Varginha|-21.58888889|-45.47333333|VAG|SBBS|0
 SBVH|Vilhena|-12.69444444|-60.09833333|BVH|SBAZ|0
 SBVT|Vitoria|-20.25805556|-40.28638889|VIX|SBRE|0
+SBWA|TMA Amazonica|-3.99957880989553|-70.089165557825496||SBAZ|0
+SBWB|TMA Belem|-1.384352664175604|-48.478670401992026||SBAZ|0
+SBWE|TMA Macae|-22.322672331759339|-40.758196984479298||SBCW|0
+SBWF|TMA Recife|-7.925945851280596|-34.961435514671074||SBRE|0
+SBWG|TMA Campo Grande|-20.469184627157627|-54.671328606785352||SBCW|0
+SBWH|TMA Belo Horizonte|-19.740666392572926|-43.944450104898209||SBBS|0
+SBWI|TMA Foz|-25.583650344677547|-54.503517780477033||SBCW|0
+SBWJ|TMA Rio de Janeiro|-22.83874572293243|-43.36042003591033||SBCW|0
+SBWK|TMA Porto Seguro|-16.496035638713362|-39.150084152734138||SBRE|0
+SBWL|TMA Ilheus|-14.760029603904959|-39.005500149632311||SBRE|0
+SBWM|TMA Santa Maria|-29.720788381771705|-53.793966794675185||SBCW|0
+SBWN|TMA Manaus|-3.036717092558909|-60.125||SBAZ|0
+SBWP|TMA Porto Alegre|-29.66766947232535|-51.170980015868949||SBCW|0
+SBWQ|TMA Boa Vista|2.852145488220792|-60.686888634166507||SBAZ|0
+SBWR|TMA Brasilia|-15.860883550633268|-47.951179304797236||SBBS|0
+SBWS|TMA Sao Luis|-2.589208924249686|-44.24||SBAZ|0
+SBWT|TMA Curitiba|-25.721198232989853|-49.040417826789536||SBCW|0
+SBWU|TMA Bauru|-22.314937528996772|-49.425650725522466||SBBS|0
+SBWV|TMA Porto Velho|-8.717141137066752|-63.9005||SBAZ|0
+SBWX|TMA Santarem|-2.426537719148094|-54.8175||SBAZ|0
+SBWY|TMA Cuiaba|-15.656258635631669|-56.112||SBAZ|0
+SBWZ|TMA Fortaleza|-3.746508258647531|-38.561163083067918||SBRE|0
+SBXA|TMA Aracaju|-11.030341351940745|-37.094902638557315||SBRE|0
+SBXB|TMA Rio Branco|-9.868178229534768|-67.89216909019818||SBAZ|0
+SBXD|TMA Palmas|-10.294729780608373|-48.36101298434027||SBBS|0
+SBXE|TMA Teresina|-5.062544020306949|-42.825133170885628||SBRE|0
+SBXF|TMA Florianopolis|-27.305777160724965|-48.56475217342247||SBCW|0
+SBXG|TMA Presidente Prudente|-22.102317082520692|-51.231373826015847||SBCW|0
+SBXJ|TMA Maraba|-5.371446074713713|-49.108488171409633||SBAZ|0
+SBXK|TMA Macapa|0.052240866690536|-51.073113871375348||SBAZ|0
+SBXL|TMA Paso de Los Libres|-29.695710272700797|-57.145668083276952||SBCW|0
+SBXM|TMA Maceio|-9.544495400271749|-35.824440752683451||SBRE|0
+SBXN|TMA Anapolis|-16.698105189562909|-49.191384856921374||SBBS|0
+SBXO|TMA Londrina|-23.398003673309756|-51.492175042027505||SBCW|0
+SBXP|TMA Sao Paulo|-23.394377654219664|-46.631045302186344||SBCW|0
+SBXQ|TMA Academia|-21.682076873497365|-48.021099658745648||SBBS|0
+SBXR|TMA Vitoria|-20.25581273483327|-40.286999943055761||SBCW|0
+SBXS|TMA Salvador|-12.737728976716769|-38.490813495620984||SBRE|0
+SBXT|TMA Natal|-5.983304017413371|-35.443228962609496||SBRE|0
+SBXU|TMA Uberaba|-19.896634471835316|-47.899767169258652||SBBS|0
+SBXW|TMA Uberlandia|-18.848263690804732|-48.174967065149005||SBBS|0

```

Comment:
Just to confirm these are not airport, but entries to allow approach sectors to display?
Can any of these be assosiated with excisting airports?
Regardless, if they are not actual airport, you must change the last digit to 1:

SBWA|TMA Amazonica|-3.99957880989553|-70.089165557825496||SBAZ|0
becomes
SBWA|TMA Amazonica|-3.99957880989553|-70.089165557825496||SBAZ|1